### PR TITLE
Added Python Example for Knative Eventing 

### PR DESCRIPTION
**Pull request by axsaucedo**
Fixes #issue-number or description of the problem the PR solves

## Proposed Changes 

Added Python Example for Knative Eventing 

The example has the exact same contents than the go example.

### CODE REVIEW

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM python:alpine3.7
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
FROM python:alpine3.7
```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM python:alpine3.7 
+
+COPY . /app
+
+WORKDIR /app
+
+RUN pip install -r requirements.txt
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
RUN pip install -r requirements.txt
```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+COPY . /app
+
+WORKDIR /app
+
+RUN pip install -r requirements.txt 
+
+EXPOSE 8080
+
+ENTRYPOINT [ "python" ]
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
ENTRYPOINT [ "python" ]
```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+WORKDIR /app
+
+RUN pip install -r requirements.txt 
+
+EXPOSE 8080
+
+ENTRYPOINT [ "python" ] 
+
+CMD [ "helloworld.py" ]
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
CMD [ "helloworld.py" ]
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+
+
+    ```
+
+1. Add a requirements.txt file containing the following contents:
+
+    ```bash
+    Flask==1.1.1
+    ```
+
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+   kubectl apply --filename sample-app.yaml
+   ```
+    1.  Above command created a namespace `knative-samples` and labelled it with `knative-eventing-injection=enabled`, to enable eventing in the namespace. Verify using the following command:
+        ```shell
+        kubectl get ns knative-samples --show-labels
+        ```
+    1. It deployed the helloworld-python app as a K8s Deployment and created a K8s service names helloworld-python. Verify using the following command.
+        ```shell
+        kubectl --namespace knative-samples get deployments helloworld-python
+
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 7]
```diff
@@ -0,0 +1,343 @@
...
+      -X POST \
+      -H "Ce-Id: 536808d3-88be-4077-9d7a-a3f162705f79" \
+      -H "Ce-specversion: 0.3" \
+      -H "Ce-Type: dev.knative.samples.helloworld" \
+      -H "Ce-Source: dev.knative.samples/helloworldsource" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello World from the curl pod."}'
+
+      exit
+      ```
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
      ```
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 8]
```diff
@@ -0,0 +1,343 @@
...
+      -H "Ce-specversion: 0.3" \
+      -H "Ce-Type: dev.knative.samples.helloworld" \
+      -H "Ce-Source: dev.knative.samples/helloworldsource" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello World from the curl pod."}'
+
+      exit
+      ``` 
+### Verify that event is received by helloworld-python app
+Helloworld-python app logs the context and the msg of the above event, and replies back with another event.
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
Helloworld-python app logs the context and the msg of the above event, and replies back with another event.
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 9]
```diff
@@ -0,0 +1,343 @@
...
+                image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display
+      ---
+      # Service that exposes event-display app.
+      # This will be the subscriber for the Trigger
+      kind: Service
+      apiVersion: v1
+      metadata:
+        name: event-display
+        namespace: knative-samples
+      spec:
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
      spec:
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 10]
```diff
@@ -0,0 +1,343 @@
...
+      spec:          
+        selector:
+          app: event-display
+        ports:
+          - protocol: TCP
+            port: 80
+            targetPort: 8080
+      END
+      ```
+
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 11]
```diff
@@ -0,0 +1,343 @@
...
+          knativehistory: default-kn2-ingress-kn-channel.knative-samples.svc.cluster.local
+          traceparent: 00-4b01db030b9ea04bb150b77c8fa86509-2740816590a7604f-00
+        Data,
+          {
+            "msg": "Hi from helloworld- app!"
+          }
+      ```
+
+  **Note: You could use the above approach to test your applications too.**
+
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile (hunk 1 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
FROM python:alpine3.7
```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile (hunk 2 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
RUN pip install -r requirements.txt
```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile (hunk 3 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
ENTRYPOINT [ "python" ]
```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile (hunk 4 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
CMD [ "helloworld.py" ]
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 5 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 6 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 7 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
      ```
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 8 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
Helloworld-python app logs the context and the msg of the above event, and replies back with another event.
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 9 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
      spec:
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 10 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 11 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile [hunk 12]
```diff
@@ -0,0 +1,13 @@
+FROM python:alpine3.7
+
+COPY . /app
+
+WORKDIR /app
+
+RUN pip install -r requirements.txt
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
RUN pip install -r requirements.txt
```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile [hunk 13]
```diff
@@ -0,0 +1,13 @@
...
+
+COPY . /app
+
+WORKDIR /app
+
+RUN pip install -r requirements.txt 
+
+EXPOSE 8080
+
+ENTRYPOINT [ "python" ]
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
ENTRYPOINT [ "python" ]
```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile [hunk 14]
```diff
@@ -0,0 +1,13 @@
...
+
+WORKDIR /app
+
+RUN pip install -r requirements.txt 
+
+EXPOSE 8080
+
+ENTRYPOINT [ "python" ] 
+
+CMD [ "helloworld.py" ]
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
CMD [ "helloworld.py" ]
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 5 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 6 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 7 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
      ```
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 8 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
Helloworld-python app logs the context and the msg of the above event, and replies back with another event.
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 9 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
      spec:
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 10 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 11 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile [hunk 15]
```diff
@@ -0,0 +1,13 @@
...
+
+COPY . /app
+
+WORKDIR /app
+
+RUN pip install -r requirements.txt
+
+EXPOSE 8080
+
+ENTRYPOINT [ "python" ]
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
ENTRYPOINT [ "python" ]
```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile [hunk 16]
```diff
@@ -0,0 +1,13 @@
...
+
+WORKDIR /app
+
+RUN pip install -r requirements.txt
+
+EXPOSE 8080
+
+ENTRYPOINT [ "python" ] 
+
+CMD [ "helloworld.py" ]
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
CMD [ "helloworld.py" ]
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 5 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 6 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 7 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
      ```
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 8 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
Helloworld-python app logs the context and the msg of the above event, and replies back with another event.
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 9 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
      spec:
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 10 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 11 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile (hunk 15 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
ENTRYPOINT [ "python" ]
```

File: docs/eventing/samples/helloworld/helloworld-python/Dockerfile (hunk 16 above)
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/Dockerfile]: Remove trailing whitespace:
```suggestion
CMD [ "helloworld.py" ]
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 17]
```diff
@@ -0,0 +1,341 @@
...
+
+    ```
+
+1. Add a requirements.txt file containing the following contents:
+
+    ```bash
+    Flask==1.1.1
+    ```
+
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 18]
```diff
@@ -0,0 +1,341 @@
...
+   kubectl apply --filename sample-app.yaml
+   ```
+    1.  Above command created a namespace `knative-samples` and labelled it with `knative-eventing-injection=enabled`, to enable eventing in the namespace. Verify using the following command:
+        ```shell
+        kubectl get ns knative-samples --show-labels
+        ```
+    1. It deployed the helloworld-python app as a K8s Deployment and created a K8s service names helloworld-python. Verify using the following command.
+        ```shell
+        kubectl --namespace knative-samples get deployments helloworld-python
+
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 19]
```diff
@@ -0,0 +1,341 @@
...
+      -X POST \
+      -H "Ce-Id: 536808d3-88be-4077-9d7a-a3f162705f79" \
+      -H "Ce-specversion: 0.3" \
+      -H "Ce-Type: dev.knative.samples.helloworld" \
+      -H "Ce-Source: dev.knative.samples/helloworldsource" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello World from the curl pod."}'
+
+      exit
+      ```
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
      ```
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 20]
```diff
@@ -0,0 +1,341 @@
...
+      -H "Ce-specversion: 0.3" \
+      -H "Ce-Type: dev.knative.samples.helloworld" \
+      -H "Ce-Source: dev.knative.samples/helloworldsource" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello World from the curl pod."}'
+
+      exit
+      ``` 
+### Verify that event is received by helloworld-python app
+Helloworld-python app logs the context and the msg of the above event, and replies back with another event.
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
Helloworld-python app logs the context and the msg of the above event, and replies back with another event.
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 21]
```diff
@@ -0,0 +1,341 @@
...
+                image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display
+      ---
+      # Service that exposes event-display app.
+      # This will be the subscriber for the Trigger
+      kind: Service
+      apiVersion: v1
+      metadata:
+        name: event-display
+        namespace: knative-samples
+      spec:
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion
      spec:
```

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 22]
```diff
@@ -0,0 +1,341 @@
...
+      spec:          
+        selector:
+          app: event-display
+        ports:
+          - protocol: TCP
+            port: 80
+            targetPort: 8080
+      END
+      ```
+
```
Comment (mattmoor) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Remove trailing whitespace:
```suggestion

```

**Comment by axsaucedo** [comment]
Thanks @mattmoor - I've approved all requested changes

**Comment by axsaucedo** [comment]
@mattmoor just wanted to see if there are still any other changes required

**Comment by mattmoor** [comment]
Ha, sorry those comments were all automated.

/assign @n3wscott 

I think @n3wscott is probably a good set of eyeballs for this.

**Comment by axsaucedo** [comment]
Haha ok thanks for the heads up  @mattmoor! @n3wscott let me know if there are any questions to merge

**Comment by axsaucedo** [comment]
Ok sounds good - @abrennan89 let me know if there are any changes required

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 23]
```diff
@@ -0,0 +1,338 @@
...
+following commands:
+
+```shell
+# Clone the relevant branch version such as "release-0.13"
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+cd knative-docs/docs/eventing/samples/helloworld/helloworld-python
+```
+
+## Before you begin
+
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Add a line to say something like "To follow this example guide, you will need:"

Also, change "we'll use it"
I'd list the Docker Hub a/c as a separate point, e.g.
- A configured Docker Hub account to use as a container registry.

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 24]
```diff
@@ -0,0 +1,338 @@
...
+        response.headers["Ce-Type"] = "dev.knative.samples.hifromknative"
+        return response
+
+    if __name__ == '__main__':
+        app.run(debug=True, host='0.0.0.0', port=8080)
+
+
+    ```
+
+1. Add a requirements.txt file containing the following contents:
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Where is this txt file added? Also, use backticks for literals? e.g. `requirements.txt`

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 25]
```diff
@@ -0,0 +1,338 @@
...
+
+    EXPOSE 8080
+
+    ENTRYPOINT [ "python" ]
+
+    CMD [ "helloworld.py" ]
+
+    ```
+
+1. Create a new file, `sample-app.yaml` and copy the following service definition
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: `sample-app.yaml`,
(add comma for readability)

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 26]
```diff
@@ -0,0 +1,338 @@
...
+          type: dev.knative.samples.helloworld
+          source: dev.knative.samples/helloworldsource
+      subscriber:
+        ref:
+          apiVersion: v1
+          kind: Service
+          name: helloworld-python
+    ```
+
+## Building and deploying the sample
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: ## Building and deploying the sample app

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 27]
```diff
@@ -0,0 +1,338 @@
...
+
+   ```shell
+   # Build the container on your local machine
+   docker build -t {username}/helloworld-python .
+
+   # Push the container to docker registry
+   docker push {username}/helloworld-python
+   ```
+
+1. After the build has completed and the container is pushed to docker hub, you
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: again, use consistent capitalization, e.g. Docker Hub

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 28]
```diff
@@ -0,0 +1,338 @@
...
+   can deploy the sample application into your cluster. Ensure that the container image value
+   in `sample-app.yaml` matches the container you built in the previous step. Apply
+   the configuration using `kubectl`:
+
+   ```shell
+   kubectl apply --filename sample-app.yaml
+   ```
+    1.  Above command created a namespace `knative-samples` and labelled it with `knative-eventing-injection=enabled`, to enable eventing in the namespace. Verify using the following command:
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: "The previous step creates...and labels it with...that enables..."

"Verify that the previous step has been successful by entering the following command:"

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 29]
```diff
@@ -0,0 +1,338 @@
...
+        ```shell
+        kubectl get ns knative-samples --show-labels
+        ```
+    1. It deployed the helloworld-python app as a K8s Deployment and created a K8s service names helloworld-python. Verify using the following command.
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: What is it? Again use present tense "X deploys...as a...and creates a..."

"Verify that the previous step has been successful by entering the following command:"

I think the structure here overall for the steps could use some work. It's confusing that each step constantly refers back to a previous step. The verification should be part of the step it's verifying maybe?

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 30]
```diff
@@ -0,0 +1,338 @@
...
+        ```shell
+        kubectl --namespace knative-samples get deployments helloworld-python
+        kubectl --namespace knative-samples get svc helloworld-python
+        ```
+    1. It created a Knative Eventing Trigger to route certain events to the helloworld-python application. Make sure that Ready=true
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Again, what is 'it'? Please modify the structure of these steps as mentioned in previous comment.

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 31]
```diff
@@ -0,0 +1,338 @@
...
+        ```shell
+        kubectl --namespace knative-samples get trigger helloworld-python
+        ```
+## Send and verify CloudEvents
+Once you have deployed the application and verified that the namespace, sample application and trigger are ready, let's send a CloudEvent.
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Once -> After
Let's -> you can

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 32]
```diff
@@ -0,0 +1,338 @@
...
+
+### Send CloudEvent to the Broker
+We can send an http request directly to the [Broker](../../../broker-trigger.md) with correct CloudEvent headers set.
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Not we.....
"You can...using the correct..."

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 33]
```diff
@@ -0,0 +1,338 @@
...
+      -H "Ce-specversion: 0.3" \
+      -H "Ce-Type: dev.knative.samples.helloworld" \
+      -H "Ce-Source: dev.knative.samples/helloworldsource" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello World from the curl pod."}'
+
+      exit
+      ```
+### Verify that event is received by helloworld-python app
+Helloworld-python app logs the context and the msg of the above event, and replies back with another event.
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: msg -> message, do not abbreviate

"The...app"

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 34]
```diff
@@ -0,0 +1,338 @@
...
+        source: knative/eventing/samples/hello-world
+        id: 37458d77-01f5-411e-a243-a459bbf79682
+      Data,
+        {"msg":"Hi from Knative!"}
+
+      ```
+  Play around with the CloudEvent attributes in the curl command and the trigger specification to understand how [Triggers work](../../../broker-trigger.md#trigger).
+
+## Verify reply from helloworld-python app
+`helloworld-python` app replies back with an event of `type= dev.knative.samples.hifromknative`, and `source=knative/eventing/samples/hello-world`. This event enters the eventing mesh via the Broker and can be delivered to other services using a Trigger
```
Comment (abrennan89) [inline docs/eventing/samples/helloworld/helloworld-python/README.md]: Again, "The...app"
Also, be consistent when using backticks, formatting, capitalization for all naming. The app is referred to in different ways throughout the doc.

Comment (abrennan89) [reply docs/eventing/samples/helloworld/helloworld-python/README.md]: Use "through", not "via"

**Comment by axsaucedo** [comment]
@abrennan89 thank you for the comments! Before diving into changing them I just want to point out that the actual content/writing is almost exactly the same as the current Golang example. There's seems to be a lot of wording / tone specific changes required here, but just want to confirm for concistency as all the comments provided here apply to your existing Golang helloworld example https://knative.dev/docs/eventing/samples/helloworld/helloworld-go/

What would be the best way forward here? Would it make sense to update the consistency of writing once there is guidelines for terminology? I can for the meantime apply the changes specific to the example such as the `requirements.txt` issue

**Comment by abrennan89** [comment]
@axsaucedo I think having some sections as close to correct as we can for now is more important than trying to have consistency across everything. We're in the process of figuring out things like style guides / glossaries for docs, so they won't be consistent for a while, but we can start with small improvements like these :slightly_smiling_face:

**Comment by abrennan89** [comment]
@axsaucedo is this PR still WIP?

**Comment by axsaucedo** [comment]
@abrennan89 sorry for the radio silence, I don't think I'll be able to get around this unfortunately, would still be worth haivng a python example tho - thanks for the comments tho @abrennan89!

**Comment by vmexalien** [comment]
working on this as part of [#2704](https://github.com/knative/docs/issues/2704)

File: docs/eventing/samples/helloworld/helloworld-python/README.md (hunk 24 above)
Comment (vmexalien) [reply docs/eventing/samples/helloworld/helloworld-python/README.md:52]: still need to determine where the requirements.txt file is added

**Comment by vmexalien** [comment]
line 148 - add comma for readbility `sample folder), you're ready`
line 150 - reword `Use Docker to build a container from the sample code.`
line 162 - correct capitalization for consistency `Docker Hub`

**Pull request merged**